From e3f4e4fd3dd3610ee8c47fc7d27812e3040e915d Mon Sep 17 00:00:00 2001
From: Birger Schacht <birger@debian.org>
Date: Sat, 4 Jan 2025 16:37:47 +0100
Subject: [PATCH] feat: comment default output setting

We comment the default output setting, because the Sway wallpaper files
are shipped in a separate package `sway-backgrounds`. Installing this
package drops a config file to @sysconfdir@/sway/config.d/
---
 config.in | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

Index: sway-regolith/config.in
===================================================================
--- sway-regolith.orig/config.in
+++ sway-regolith/config.in
@@ -23,7 +23,10 @@ include /etc/sway/config-vars.d/*
 ### Output configuration
 #
 # Default wallpaper (more resolutions are available in @datadir@/backgrounds/sway/)
-output * bg @datadir@/backgrounds/sway/Sway_Wallpaper_Blue_1920x1080.png fill
+# This is commented in Debian, because the Sway wallpaper files are in a separate
+# package `sway-backgrounds`. Installing this package drops a config file to
+# @sysconfdir@/sway/config.d/
+# output * bg @datadir@/backgrounds/sway/Sway_Wallpaper_Blue_1920x1080.png fill
 #
 # Example configuration:
 #
